interface-definitions/include/firewall/match-vlan.xml.i

```diff
@@ -0,0 +1,41 @@
+<!-- include start from firewall/match-vlan.xml.i -->
+<node name="vlan">
+  <properties>
+    <help>VLAN parameters</help>
+  </properties>
+  <children>
+    <leafNode name="id">
+      <properties>
+        <help>Vlan id</help>
```

Comment:
VLAN should be capitalized, since it's an abbreviation (Virtual Local Area Network),

Reply:
@dmbaturin does ID should be also capitalized :)